src/test/common/steps/guidance-steps.js

```diff
@@ -0,0 +1,41 @@
+const { When, Then } = require('cucumber')
+const { expect } = require('chai')
+
+const pages = require('./pages')
+
+When(/^I open the (.*) guidance page$/, async function (pageName) {
+  await pages.guidance.openGuidancePage(pages.url, pageName)
+})
+
+Then(/^all the (.*) guidance page content is present$/, async function (pageName) {
+  await assertGuidancePageHeadersCorrect(pageName)
+  await assertGuidancePageContentsCorrect(pageName)
+  await assertGuidancePageNavigationLinksCorrect(pageName)
+})
+
+async function assertGuidancePageHeadersCorrect (pageName) {
+  const h1Text = await pages.cookies.getH1Text()
+  expect(h1Text.toString().trim()).to.be.equal('Get money off milk, food and vitamins (Healthy Start)',
```

Comment:
is the H1 text the same on every guidance page?

Reply:
Yes.